### PR TITLE
Added vg tools to new cgl-docker-lib/vg directory.

### PR DESCRIPTION
**Pull request by cmarkello**

### CODE REVIEW

File: vg/runtime/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+
+MAINTAINER Charles Markello, cmarkell@ucsc.edu
+
+RUN mkdir /opt/vg/
+
+RUN pwd
+RUN ls -l
```
Comment (hannes-ucsc) [inline vg/runtime/Dockerfile]: I'd probably remove these two. Unless you have a good reason for leaving them in. I see how they might help during initial development but I don't see their value at this point.

File: vg/runtime/wrapper.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```
Comment (hannes-ucsc) [inline vg/runtime/wrapper.sh:1]: Just, FYI, these will go away. @jvivian is working on this. John, we should bite the bullet and remove all of them at once, don't you think?

Comment (hannes-ucsc) [inline vg/runtime/wrapper.sh:1]: And by these I mean `wrapper.sh`.

**Comment by cmarkello** [comment]
Ok I removed the debugging lines from the runtime dockerfile. I forgot to remove them.

Also, not sure what you mean by the wrapper.sh being removed. Isn't it necessary for user interaction with the tool?

**Comment by hannes-ucsc** [comment]
For manual invocations, we'll probably ignore the permissions issue that wrapper.sh works around. For invocations in Toil pipelines we'll augment our generic docker_call function to fix the permissions.

https://github.com/BD2KGenomics/toil-scripts/pull/295

**Comment by hannes-ucsc** [comment]
+1 from me. @jvivian or @fnothaft any objections?

**Comment by fnothaft** [comment]
+1

**Comment by jvivian** [comment]
+1

**Comment by hannes-ucsc** [comment]
Jenkins, test this please.

**Comment by hannes-ucsc** [comment]
Ahh, we don't have PR builder enabled on this. Never mind ...

**Pull request merged**